docs/design/fabric-v2+/fpc-lifecycle-v2-new.puml

```diff
@@ -0,0 +1,139 @@
+/'
+    Copyright 2020 Intel Corporation
+    Copyright IBM Corp. All Rights Reserved.
+
+    SPDX-License-Identifier: Apache-2.0
+'/
+
+@startuml
+
+!pragma teoz true
+
+hide footbox
+title Foot Box removed
+
+title Fabric Private Chaincode - Lifecycle v2
+
+box "Org1"
+actor	Admin order 10
+entity "FPC_Peer1_Wrapper" as FPC_Peer1_Wrapper order 15
+participant Peer1 order 20
+entity "FPC_Peer2_Wrapper" as FPC_Peer2_Wrapper order 30
+participant Peer2 order 40
```

Comment:
our current implementation calls this a peer wrapper, however, when we talk about the new design should the e_manager be involved in the action that the wrapper performs here now?

Reply:
Or should it be just a FPC_PEER? the wrapper is an actual implementation detail from my perspective

Reply:
the peer wrapper is really the peer _cli_ wrapper and from what we discussed on the other diagram maybe we should just call it `cli`?